packages/jellyfish-api-core/__tests__/category/wallet/signMessage.test.ts

```diff
@@ -0,0 +1,41 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Sign Message', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should throw error if private key is not available', async () => {
+    const promise = client.wallet.signMessage('mpLQjfK79b7CCV4VMJWEWAj5Mpx8Up5zxB', 'This is a test message')
```

Comment:
```suggestion
    const address = await client.wallet.getNewAddress()
    const promise = client.wallet.signMessage(address, 'This is a test message')
```

This is how you create a new address within the Wallet and then use it for signing.